core/components/atoms/select/select.js

```diff
@@ -20,14 +22,14 @@ const Select = ({ options, ...props }) => {
   return (
     <StyledSelect {...props} {...Automation('select')}>
       {/* First option will be selected if there is no value passed as a prop */}
-      <option disabled hidden selected={!props.value} value="" {...Automation('select.option')}>
+      <StyledOption disabled hidden selected={!props.value} value="" {...Automation('select.option')}>
         {props.placeholder}
-      </option>
+      </StyledOption>
 
       {options.map((option, index) => (
-        <option key={index} value={option.value} {...Automation('select.option')}>
+        <StyledOption key={index} value={option.value} readOnly={option.readOnly} disabled={option.readOnly} {...Automation('select.option')}>
```

Comment:
Err why?

Reply:
the readOnly styling can be implemented this way. Tried css selectors, but didn't work.

Reply:
We can just use the `disabled` attribute of the `<select />`

Reply:
[here](https://github.com/auth0/cosmos/issues/1062#issuecomment-435567902) I discussed the hardly noticeable difference of the disabled select option

Reply:
We need to use the disabled attribute either way for accessibility, I think we can add our custom styles to make it look more disabled 🙂

Reply:
Fixed it :)